### PR TITLE
MOD-2599 - UT: added unit test for all the functions.

### PR DESCRIPTION
**Pull request by anil-shukla-axway**
https://jira.appcelerator.org/browse/MOD-2599

### CODE REVIEW

File: test/unit/specs/module.spec.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+			it('QUALITY_MEDIUM', () => {
+				expect(imagefactory.QUALITY_MEDIUM).toEqual(jasmine.any(Number));
+			});
+
+			it('QUALITY_HIGH', () => {
+				expect(imagefactory.QUALITY_HIGH).toEqual(jasmine.any(Number));
+			});
+		});
+	});
+		describe ('hasAlpha' , () =>{
```
Comment (fasihul-axway) [inline test/unit/specs/module.spec.js:71]: @anil-shukla-axway, create a describe for methods `describe('methods', () => {}` and then write methods UT inside it.

**Comment by aakash-jain-axway** [comment]
@anil-shukla-axway : PR title should start with UT like title in [PR](https://github.com/appcelerator-modules/ti.imagefactory/pull/37) instead of feat(ios)

File: test/unit/specs/module.spec.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+				it('imgage should has an alpha layer',  () =>{
```
Comment (aakash-jain-axway) [inline test/unit/specs/module.spec.js:72]: change to "should have". same for other places as well.

**Comment by fasihul-axway** [comment]
@anil-shukla-axway, fix linting issue, jenkins checks are failing.

File: test/unit/specs/module.spec.js (hunk 2 above)
Comment (anil-shukla-axway) [reply test/unit/specs/module.spec.js]: Done

File: test/unit/specs/module.spec.js (hunk 1 above)
Comment (anil-shukla-axway) [reply test/unit/specs/module.spec.js]: Done

**Comment by fasihul-axway** [comment]
@anil-shukla-axway, 7 test cases are failing. 

Screenshot 2020-05-08 at 1 48 58 PM

**Pull request merged**